lib/pluginUtils.js

```diff
@@ -25,12 +25,15 @@ function getPluginName (func) {
   // let's see if this is a file, and in that case use that
   // this is common for plugins
   const cache = require.cache
-  const keys = Object.keys(cache)
-
-  for (let i = 0; i < keys.length; i++) {
-    const key = keys[i]
-    if (cache[key].exports === func) {
-      return key
+  // cache is undefined inside SEA
```

Comment:
What is SEA? This comment is mysterious.

Reply:
South East Asia, obviously.

Reply:
"Single executable application" sounds also valid, but I prefer south east asia.